ARCHITECTURE.md

```diff
@@ -0,0 +1,83 @@
+## Architecture
+
+- Survey MoonBear is a Ruby Web application.
+- During survey-taking, browser javascript acts like an SPA (single page application).
+    - Next page button just changes the view in javascript.
+    - Only when respondent submits the survey at the end, browser javascript POSTs the filled survey to App.
+
+### Services
+*which service corresponds to which function of the application?*
+1. Auth
+    - `FindAuthenticatedGoogleAccount`: service to return  an authenticated user, or nil. e.g. id, email, username, access_token
+2. Outputs
+    - `TransformResponsesToCSV`: service to transform Launch.responses to CSV file
+    - `TransformDBSurveyToHTML`: service to create HTML strings: title & array of each survey page
+    - `TransformSheetsSurveyToHTML`: service to create HTML strings: title & array of each survey page
+3. Responses
+    - `StoreResponses`: send message(responses_hash) to queue
+4. Retrieve Surveys
+    - `GetSurveyFromDatabase`: return an entity of survey from database
+    - `GetSurveyFromSpreadsheet`: return an entity of survey from spreadsheet
+5. Surveys
```

Comment:
Rename it to "Manage Surveys"

Reply:
Move everything except start/close to a section called "Modify Surveys"

Reply:
We done~~